helm/cluster-openstack/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: cluster-openstack
 description: A helm chart for creating Cluster API clusters with the OpenStack infrastructure provider (CAPO).
 home: https://github.com/giantswarm/cluster-openstack
 type: application
-version: 0.17.0
+version: 0.17.1
```

Comment:
don't update manually. will be updated by ci jobs.

Reply:
Remove the manual update.